android/src/main/java/com/clevertap/react/CleverTapUtils.java

```diff
@@ -1,11 +1,15 @@
 package com.clevertap.react;
 
+import android.os.Handler;
```

Comment:
All the imports added in this file are redundant

Reply:
fixed